src/main/resources/dashboard/user.hbs

```diff
@@ -0,0 +1,54 @@
+{{#partial "content"}}
+
+<p>Obtain
+GitHub <a href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token">personal access token</a>
+and use it as a password in authentication through package managers,
+your username is <code>github.com/{{user}}</code>.</p>
+
+<!--
+<form action="/api/users/{{user}}/password" method="POST">
+  <input id="new-password" name="password" type="password" placeholder="Your password..." size="35" required="true"/>
+  <input id="new-repo-submit" type="submit" value="Change"/>
+</form>
+-->
```

Comment:
@olenagerasimova I understand that it is not scope of the PR, but let's remove this commented out part